packages/lib-utils/src/components/table/Table.tsx

```diff
@@ -0,0 +1,126 @@
+import type { ICell, SortByDirection, ThProps } from '@patternfly/react-table';
+import { TableComposable, Tbody, Th, Thead, Tr } from '@patternfly/react-table';
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import type { LoadError } from '../status/StatusBox';
+import { StatusBox } from '../status/StatusBox';
+
+export type RowProps<D> = {
+  obj: D;
+};
+
+export type TableColumn<D> = ICell & {
+  title: string;
+  id: string;
+  sort?: ThProps['sort'] | ((data: D[], sortDirection: SortByDirection) => D[]) | string;
+};
+
+type TableProps<D> = {
```

Comment:
@fhlavac we should also export the table props object to be used when creating an instance of the table.